
Needed for util-linux/more, huh?

  - Rene Rebe <rene@exactcode.de>

--- ./include/a.out.h	2005-12-29 16:21:21.231999000 +0000
+++ ./include/a.out.h	2006-01-01 10:38:32.000000000 +0000
@@ -0,0 +1,27 @@
+#ifndef __A_OUT_GNU_H__
+#define __A_OUT_GNU_H__
+
+enum machine_type
+{
+  M_OLDSUN2 = 0,
+  M_68010 = 1,
+  M_68020 = 2,
+  M_SPARC = 3,
+  M_386 = 100,
+  M_MIPS1 = 151,
+  M_MIPS2 = 152
+};
+
+/* Code indicating object file or impure executable.  */
+#define OMAGIC 0407
+/* Code indicating pure executable.  */
+#define NMAGIC 0410
+/* Code indicating demand-paged executable.  */
+#define ZMAGIC 0413
+/* This indicates a demand-paged executable with the header in the text. 
+   The first page is unmapped to help trap NULL pointer references.  */
+#define QMAGIC 0314
+/* Code indicating core file.  */
+#define CMAGIC 0421
+
+#endif /* __A_OUT_GNU_H__ */
